test_strategy.md

```diff
@@ -40,3 +40,13 @@ coverage
 * Each PR changing implementation logic targeting the
 [network](https://github.com/openstack-k8s-operators/infra-operator/tree/main/controllers/network)
 controllers have to include relevant EnvTest coverage.
+
+### lib-common
```

Comment:
Ack and +1 on my side. 
FWIW when we started the ceph module at the time (just helpers, not much used in the operators ecosystem) we added something like [1], which seems to fit the purpose.
I'm pretty sure we can improve the `storage/` module and add test cases for the existing functions, but in general for functional tests I would defer to the operators implementing the feature to cover the `scenario` and double check (via `envTest`) the expected results.
@gibizer how does it sounds?

[1] https://github.com/fmount/lib-common/blob/extra_volumes/modules/storage/ceph/funcs_test.go

Reply:
Cool I will update the patch. I looked through the storage module and I don't see any k8s API calls in there so the existing unit test feel OK to me as is. We can discuss any incoming new features in that module separately but I will assume for now that unit test coverage will suffice.

Reply:
Ohh I already have the unit / functional distinction in the current docs patch. So I think both the storage and ansible module is belonging to the unit tested category.